oneview_redfish_toolkit/blueprints/network_interface_collection.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (2017) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+from flask import abort
+from flask import Blueprint
+from flask import Response
+from flask_api import status
+
+from hpOneView.exceptions import HPOneViewException
+from oneview_redfish_toolkit.api.network_interface_collection \
+    import NetworkInterfaceCollection
+
+from oneview_redfish_toolkit import util
+
+import logging
+
+network_interface_collection = Blueprint(
+    "network_interface_collection", __name__)
+
+
+@network_interface_collection.route(
+    "/redfish/v1/Systems/<uuid>/NetworkInterfaces", methods=["GET"])
```

Comment:
"/redfish/v1/Systems/<uuid>/NetworkInterfaces" change to "/redfish/v1/Systems/<uuid>/NetworkInterfaces/"